### DISCUSSION TITLE
Mount gives back no such file or directory 2.35.1

### DISCUSSION THREAD

**Discussion by DanAFC** [trapexit/mergerfs#1174] [General]
Hey, I'm new to mergerfs so looking to get started and having a bit of trouble. Aiming to setup the most basic merge.

**Problem**
Attempting to merge 2 drives (although here testing with 1, mnt/drive1) and to have that pool into say media/newdata

Before running mergerfs from the command line the pool dir (`ls -lah /media`):
`drwxr-xr-x  2 dan  dan  4.0K Apr 14 09:53 newdata`

and `ls -lah /mnt`
`drwxr-xr-x  4 dan  dan  512K Apr 14 09:27 disk1`

After running 
`sudo mergerfs -o allow_other,use_ino,cache.files=partial,dropcacheonclose=true,category.create=mfs /mnt/drive1 /media/newdata`

(tried without allow_other and without  given the docs mention that is no longer needed in v2.35.1)

There is no message after running this command but when ls -lah /media it shows
`d?????????  ? ?    ?       ?            ? newdata`

which I cannot access, when cd into that folder I'm told there is no such file or directory. The same behaviour occurs when using `/etc/fstab `instead

In `/etc/fuse.conf` I have tried uncommenting `user_allow_other `but still to no avail.

Appreciate any help and insight you may have. Thank you!

`mergerfs -v
mergerfs version: 2.35.1`

Appears similar in nature to this one but really not sure: https://github.com/trapexit/mergerfs/discussions/1138

**Comment by trapexit** [comment]
You say above that the branch directory path is `/mnt/disk1` but the path you use when mounting is `/mnt/drive1`. If the paths provided don't exist it skips them. When it skips them it has nothing to report to the system. You have to have a path that exists.

**Reply by DanAFC** [reply]
Oh wow, thanks for the rapid response! 

And yep, that was exactly it so sorry. I was expecting some form of logging/message but that behaviour makes sense also. Thanks @trapexit

**Reply by trapexit** [reply]
It will log to syslog if you use the `branches-mount-timeout` feature. 

Part of the problem is that mergerfs accepts globs. All branches have to be interpreted as such. And a glob can return nothing. Also mergerfs doesn't really care about the existence and that is useful in certain situations. Allows you to remove a branch on the fly if needed. In v3 I will be adding more context to branch definitions so it can be smarter about how it responds but right now the logic people expect is too complicated to articulate in the supported config language.

**Reply by trapexit** [reply]
The reason that feature will lead to syslog entries is because it's a new feature and I just added syslog'ing. I will add more as I recognize the need. Like here I could simply indicate that nothing was found and skipped. I also plan on adding a "nothing at all was found" message.

**Reply by DanAFC** [reply]
Incredible, thanks for the detailed answer and you're well ahead of me given you've already thought about that. Great project, appreciate your work on it!
